src/redux/mission/mission.js

```diff
@@ -1,11 +1,28 @@
+/* eslint-disable no-console */
```

Comment:
Hello @IfzaRasool just like the one above kindly remove this line of your code, it might make your code-base less readable, and also your application could take more time to load